backend/database/_client.py

```
@@ -2,27 +2,99 @@
 import json
 import os
 import uuid
-
 from google.cloud import firestore
+from google.cloud.firestore_v1 import AsyncClient
+from google.oauth2 import credentials
```

Ruff

`google.oauth2.credentials` imported but unused

Remove unused import: `google.oauth2.credentials`

(F401)